src/HttpTrigger/tests/index.test.js

```diff
@@ -0,0 +1,12 @@
+var chai = require('chai');
```

Comment:
Since you're using `const`, why not use `let` instead of `var`?

Reply:
Done.